mints/mapping.txt

```diff
@@ -0,0 +1,17 @@
+token_id-"component_id"
```

Comment:
@gauravlochab and @Divya-Solulab are there any packages missing from this list that are part of optimus and in this repo (i.e. not external dependencies)?

Reply:
No, all the packages are there.

Reply:
No